### ISSUE TITLE
Can you release the code for automatic instruction data generation?

### ISSUE THREAD

**Issue by JonghwanMun** [jihaonew/MM-Instruct#2]
First, thank you for the great work :)
I wonder do you have a plan to release the instruction data generation code.

**Comment by jihaonew** [comment]
Yes, we are working on releasing the data generation code. Please stay tuned.
